main.go

```diff
@@ -0,0 +1,195 @@
+	IncreaseMonsterDensity int           `json:"IncreaseMonsterDensity"`
+	EnableTownSkills       bool          `json:"EnableTownSkills"`
+	NoDropZero             bool          `json:"NoDropZero"`
+	QuestDrops             bool          `json:"QuestDrops"`
+	UniqueItemDropRate     int           `json:"UniqueItemDropRate"`
+	StartWithCube          bool          `json:"StartWithCube"`
+	RandomOptions          RandomOptions `json:"RandomOptions"`
+}
+
+type RandomOptions struct {
+	Randomize    bool `json:"Randomize"`
+	Seed         int  `json:"Seed"`
+	IsBalanced   bool `json:"IsBalanced"`
+	MinProps     int  `json:"MinProps"`
+	MaxProps     int  `json:"MaxProps"`
+	UseOSkills   bool `json:"UseOSkills"`
+	PerfectProps bool `json:"PerfectProps"`
+}
+
+//PageVariables - GUI variables that change on webpages.
+type PageVariables struct {
+	Output string
+}
+
+//Global struct to save the settings.
+var N = Settings{}
+
+//Main page for the front end.
+var mapage = `<title>d2modmaker config editor</title>
+</head>
+<body>
+    <style>
+        input {
+            display: inline-block;
+            float: left;
+            margin-right: 20px;
+			background-color: #ffc2c2
+		}
+    </style>
+    <a href="https://imgbb.com/"><img src="https://i.ibb.co/kxScnHd/diablo2.jpg" alt="Diablo II Logo (small)" border="0"></a>
+    <p><b>d2modmaker config editor</b></p>
+    <p>View and edit the cfg.json file below...</p>
+  
+  
+</body>
+<form action="/process" method="POST">
+	IncreaseStackSizes         <input name="IncreaseStackSizes" type="text" value="{{.IncreaseStackSizes}}" /> <br> </br>
+	IncreaseMonsterDensity     <input name="IncreaseMonsterDensity" type="text" value="{{.IncreaseMonsterDensity}}" /> <br> </br>
+	EnableTownSkills           <input name="EnableTownSkills" type="text" value="{{.EnableTownSkills}}" /> <br> </br>
+	NoDropZero                 <input name="NoDropZero" type="text" value="{{.NoDropZero}}" /> <br> </br>
+	QuestDrops                 <input name="QuestDrops" type="text" value="{{.QuestDrops}}" /> <br> </br>
+	UniqueItemDropRate         <input name="UniqueItemDropRate" type="text" value="{{.UniqueItemDropRate}}" /> <br> </br>
+	StartWithCube              <input name="StartWithCube" type="text" value="{{.StartWithCube}}" /> <br> </br>
+	RandomOptions.Randomize    <input name="Randomize" type="text" value="{{.RandomOptions.Randomize}}" /> <br> </br>
+	RandomOptions.Seed         <input name="Seed" type="text" value="{{.RandomOptions.Seed}}" /> <br> </br>
+	RandomOptions.IsBalanced   <input name="IsBalanced" type="text" value="{{.RandomOptions.IsBalanced}}" /> <br> </br>
+	RandomOptions.MinProps     <input name="MinProps" type="text" value="{{.RandomOptions.MinProps}}" /> <br> </br>
+	RandomOptions.MaxProps     <input name="MaxProps" type="text" value="{{.RandomOptions.MaxProps}}" /> <br> </br>
+	RandomOptions.UseOSkills   <input name="UseOSkills" type="text" value="{{.RandomOptions.UseOSkills}}" /> <br> </br>

```

Comment:
i think bools should be checkboxes.
int / float fields could be range sliders since we know the min/max values